projects/cdk/schematics/ng-update/v4/steps/constants/modules-to-remove.ts

```diff
@@ -21,6 +21,10 @@ export const MODULES_TO_REMOVE: RemovedModule[] = [
         name: 'TuiPromptModule',
         moduleSpecifier: '@taiga-ui/kit',
     },
+    {
+        name: 'TuiPdfViewerModule',
```

Comment:
`TuiPreviewActionModule` should also be removed.

Reply:
Does it? Shouldn't it be changed to `TuiPreview` instead since it's about buttons used in template, not about providers?

Reply:
fixed

Reply:
@waterplea the `TuiPreviewModule` should be replaced by `TuiPreview`, but `TuiPreviewActionModule` should be removed entirely since it's not used at all anymore. 

@splincode  `TuiPreviewActionModule` should be removed from NgModules and standalone component imports. Not from `importProvidersFrom()`. `TuiPreviewActionModule` should be part of `modules-to-remove`.